.github/pull_request_template.md

```diff
@@ -0,0 +1,48 @@
+## PR Type
+What kind of change does this PR introduce?
+
+- [ ] Bugfix
+- [ ] Feature
+- [ ] Code style update (formatting, local variables)
+- [ ] Refactoring (no functional changes, no api changes)
+- [ ] Build related changes
+- [ ] CI related changes
+- [ ] Documentation content changes
+- [ ] Tests
+- [ ] Other
```

Comment:
Would updating dependencies be included in `Other`? Or should we have it as a separate option? For example, I find myself updating the `@standardnotes/features` package very often with new editors/themes versions.

Reply:
I think it can fall under `Other`